core/src/main/scala/kafka/log/Log.scala

```diff
@@ -103,20 +103,22 @@ class Log(val dir: File,
     else
       0
   }
-  val t = time.milliseconds
+
+  private[this] val t = time.milliseconds
```

Comment:
I looked into changing this from a val to a local before settling for `private[this]` and my preferred solution required more changes than I would like to include in this PR. Thinking about it some more, there is a simpler change that I think is OK to include so doing that.